rim/models.py

```diff
@@ -5,11 +5,11 @@
 
 class Equipment(models.Model):
     serial_no = models.CharField(max_length=100, verbose_name='Serial Number')
-    equipment_model = models.CharField(max_length=30)
+    equipment_model = models.CharField(max_length=30, verbose_name='Equipment Model')
```

Comment:
These verbose_names are kind of unnecessary. A more generic approach would make use of the [`pretty_name()` helper function](https://github.com/django/django/blob/stable/2.1.x/django/forms/utils.py#L11-L15) (from django.forms.utils) in the view.